nservicebus/outbox/index.md

```diff
@@ -15,7 +15,11 @@ related:
 - persistence/ravendb/outbox
 ---
 
-Outbox allows endpoints to run with similar reliability to DTC while not actually using DTC.
+The Outbox is an infrastructure feature designed to simulate the reliability of distributed transactions (using the Distributed Transactions Coordinator or DTC) while not actually using the DTC.
+
+The purpose of the DTC is to guarantee consistency between messaging operations and the data persistence. Messaging operations includes the message being processed as well as any messages being sent out as a result. Data persistence includes any business data persisted by a message handler, as well as any NServiceBus saga or timeout data stored at the same time. The DTC ensures that all these operations either complete successfully together, or are all rolled back.
+
+Instead of the DTC, the NServiceBus Outbox feature can be used to mimic this level of consistency without the need for distributed transactions. It does this by first storing any outgoing messages in the database, taking advantage of the same (non-distributed) local transaction to ensure that the messaging operations are stored atomically with any changes to business data and NServiceBus data. Once that transaction is successfully committed, the stored outgoing messages are dispatched to the message transport separately.
```

Comment:
The phrase "message transport" seems off. I don't think we generally refer to transports that way elsewhere. Perhaps "dispatched to their destinations", or maybe just "dispatched."